packages/tldraw/src/lib/ui/components/Toasts.tsx

```diff
@@ -80,36 +82,16 @@ function Toast({ toast }: { toast: TLUiToast }) {
 	)
 }
 
-function _Toasts() {
-	const { toasts } = useToasts()
-
-	return toasts.map((toast) => <Toast key={toast.id} toast={toast} />)
-}
-
-export const Toasts = React.memo(_Toasts)
-
-export function ToastViewport() {
-	const editor = useEditor()
+/** @public @react */
+export const TldrawUiToasts = memo(function TldrawUiToasts() {
 	const { toasts } = useToasts()
-
-	const [hasToasts, setHasToasts] = React.useState(false)
-
-	React.useEffect(() => {
-		let timeoutId = -1
-		if (toasts.length) {
-			setHasToasts(true)
-		} else {
-			timeoutId = editor.timers.setTimeout(() => {
-				setHasToasts(false)
-			}, 1000)
-		}
-
-		return () => {
-			clearTimeout(timeoutId)
-		}
-	}, [toasts.length, setHasToasts, editor])
-
-	if (!hasToasts) return null
```

Comment:
this is very much legacy from a time when I was trying to animate toasts out of the viewport